### DISCUSSION TITLE
Eslint

### DISCUSSION THREAD

**Discussion by mharj** [community/community#17001] [Codespaces]
On normal devcontainer eslint VSCode plugin works ok, but some reason I can't get one working with Codespaces?
Works ok on command line though.
Oh and this is fast!👍

**Comment by dmilburn** [comment]
Hi from the Codespaces team! I'm sorry you're having trouble with the plugin! Could you please describe more what's not working? Like are you unable to install it? Is it not running automatically like you'd expect?

Also just to double check, are your versions of VSCode, the eslint plugin and the codespaces extension up to date?

**Reply by mharj** [reply]
was running on VSC  insider, I'm usually checking that eslint starts correctly from Ouput and select Eslint on list. But some reason eslint was not on list. Anyway I'll double check in codespace later again as I did check that devcontainer was ok like also running normally.

**Reply by mharj** [reply]
with VSC 1.67.1 works nicely, I'll check with insider later if that was causing or was it just temporary issue on VSC.

**Reply by mharj** [reply]
works ok also on latest insider, no clue why didn't start on first time 🤷‍♂️
